configure-pull-request/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Configure Pull Request
+
+description: Sets the parameters of an existing Pull Request at once
+
+inputs:
+  github-token:
+    description: A GitHub personal access token with write access to the project
+    required: true
+  labels:
+    description: Comma-separated list of labels
+    required: false
+  project-url:
+    description: URL of a GitHub project
+    required: false
+  reviewers:
+    description: Comma-separated list of users
+    required: false
+  team-reviewers:
+    description: Comma-separated list of teams
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Set labels
+      uses: buildsville/add-remove-label@eeae411a9be2e173f2420e1644514edbecc4e835 # pin@v2.0.0
+      with:
+        labels: ${{ inputs.labels }}
+        token: ${{ inputs.github-token }}
+        type: add
+      if: ${{ inputs.labels != '' }}
```

Comment:
I would suggest moving the `if` condition right under `name` so it's immediately clear that this step is conditional